test/02_gsn_acl_factory.js

```diff
@@ -0,0 +1,117 @@
+require('dotenv').config({ path: `${__dirname}/.env` });
+
+const chai = require('chai');
+chai.use(require('chai-as-promised'));
+const assert = chai.assert;
+
+const { fundRecipient } = require('@openzeppelin/gsn-helpers');
+const { fromConnection, useEphemeralKey } = require('@openzeppelin/network');
+const { utils } = require('@openzeppelin/gsn-provider');
+const { isRelayHubDeployedForRecipient, getRecipientFunds } = utils;
+
+const ACLFactory = artifacts.require("AclFactory");
+const ACL = artifacts.require("Acl");
+
+contract('AclFactory', (accounts) => {
+  const ROOT_ACCOUNT = process.env.NETWORK === 'ganache' ? accounts[0] : process.env.ACCOUNT;
+  const RELAY_HUB = process.env.RELAYHUB;
+  // Factory will be able to create 5 new individual ACLs
+  const GAS_PRICE = '500000000000';
```

Comment:
[Minor] In order to make our code more escalable you can use web3 [getGasPrice](https://web3js.readthedocs.io/en/v1.2.0/web3-eth.html#getgasprice) to obtain the correct value automatically.

Reply:
Ok but it's a super microoptimization heh... Given is hardcoded in many places like Truffle etc anyway.

Reply:
Yep, I know, that is the reason for the minor